src/instructlab/utils.py

```diff
@@ -239,6 +242,71 @@ def git_clone_checkout(
     return repo
 
 
+def get_confluence_docs(
+    ctx,
+    source: Dict[str, Union[str, List[str]]],
+) -> List[str]:
+    """
+    Retrieve the content of Confluence documents
+
+    Args:
+        source (dict): Source info containing references
+
+    Returns:
+         List[str]: List of document contents.
+    """
+    # pylint: disable=C0415
```

Comment:
Use human-readable `import-outside-toplevel` flag instead of the code. I know parts of the code are using the code but I believe we trying move away from this. Thanks!